ingesters/preprocessors/preprocessors.md

```diff
@@ -545,3 +552,89 @@ For this example we are using the Gravwell Federator to forward subsets of logs
 
 
 ```
+
+## Gravwell Forwarding Preprocessor
+
+The Gravwell forwarding processor allows for creating a complete Gravwell muxer which can duplicate entries to multiple instances of Gravwell.  This preprocessor can be useful for testing or situations where a specific Gravwell data stream needs to be duplicated to an alternate set of indexers.  The Gravwell forwarding preprocessor utilizes the same configuration structure to specify indexers, ingest secrets, and even cache controls as the package ingesters.  The Gravwell forwarding preprocessor is a blocking preprocessor, this means that if you do not enable a local cache it can block the ingest pipeline if the preprocessor cannot forward entries to the specified indexers.
+
+The Gravwell Forwarding preprocessor type is `gravwellforwarder`.
+
+### Supported Options
+
+See the [Global Configuration Parameters](#!ingesters/ingesters.md#Global_Configuration_Parameters) section for full details on all the Gravwell ingester options.  Most global configuration blocks from any production ingester configurations in the Gravwell Forwarder preprocessor.
```

Comment:
Last sentence in this paragraph isn't a complete sentence... how about "Most global ingester configuration options are supported by the Gravwell Forwarder preprocessor."?